common/static/sass/edx-pattern-library-shims/_buttons.scss

```diff
@@ -0,0 +1,121 @@
+// ------------------------------
```

Comment:
This is css v1. We have UXPL in v2. We can't use v2 here, the layout will break. That's why this comes the shim folder in the v1.